src/style/app-fixed.module.css

```diff
@@ -4020,30 +4020,160 @@
 
 /* -- GeneralSettings.tsx -- */
 
+.cardSettings {
+  border-radius: 0.5rem;
+  margin-bottom: 1.5rem;
+  border: none;
+  box-shadow: 0 0.125rem 0.25rem rgba(0, 0, 0, 0.075);
+  height: 800px;
+  width: 588px;
+}
+
+.cardHeaderStyle {
+  border-top-left-radius: 0.25rem;
+  border-top-right-radius: 0.25rem;
+  padding: 1.25rem 1rem 1rem 1rem;
+  border-bottom: 3px solid var(--cardHeader-border);
+  background-color: #eaebef;
```

Comment:
This color is defined previously in the file. Use the appropriate variable or create a new one.

Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`